pkg/install/linux_openrc.go

```diff
@@ -23,7 +23,7 @@ name="{{.DisplayName}}"
 description="{{.Description}}"
 command={{.Path|cmdEscape}}
 {{- if .Arguments }}
-command_args="{{range .Arguments}}{{.}} {{end}}"
+command_args="{{range .Arguments}}'{{.}}' {{end}}"
```

Comment:
`--foo=bar baz,--bar=5` becomes `'--foo=bar baz' '--bar=5'`, looks weird but should work